.github/workflows/onPushToMain.yml

```diff
@@ -3,13 +3,18 @@ name: version, tag and github release
 
 on:
   push:
-    branches: [main]
+    branches:
+      - main
+      - prerelease/*
+    tags-ignore:
+      - "*"
 
 jobs:
   release:
-    uses: oclif/github-workflows/.github/workflows/githubRelease.yml@main
+    uses: oclif/github-workflows/.github/workflows/githubRelease.yml@prereleases-semver
     secrets: inherit
-
+    with:
+      prerelease: ${{ github.ref_name != 'main' }}
```

Comment:
This would cause any commit that's on a `prerelease/*` branch to trigger, which could be a problem if someone makes a `fix:/feat:` commit to a  prerelease branch without a valid prerelease identifier in the `package.json`'s `version` because the [npm publish workflow below](https://github.com/oclif/plugin-autocomplete/pull/365/files#diff-07ad11b4716ea9cf0395ab663324e2ec514bd862242160a44ce12856afcc87fcR30) defaults to `latest` as a tag.

So, what is intended as a prerelease could default to the `latest` dist-tag on npm.